src/contradictory_claims/data/process_claims.py

```diff
@@ -114,14 +117,22 @@ def pair_similar_claims(claims_data: pd.DataFrame, nlp):
     # Add a new column for storing the drug terms present in each claim
     claims_data['drug_terms_mention'] = [[d for d in drug_terms if d in c] for c in claims_data.claims]
```

Comment:
For a given claim, are the drug term mentions coming from the entire paper, from any claim in that paper, or from that specific claim? I ask because I did one additional filter to check that the claim1 and claim2 text have a common drug mention, where I'm looking for a drug mention in a custom regex. And iirc, I was filtering out some claim pairs where the two claims don't have a common drugs mentioned in the text (although they might have an overlap in the drug_mentions column).

Reply:
So originally in Michael's code, the drug_terms_mentions column had the drugs present in the section from which the claims were extracted (comment in line #107), and the processing of earlier versions (including v31 , if I'm not wrong) were based on this.
While refactoring the code, I added this step to (line #118) so that when pairing, we're only comparing the drug names present in the claim sentence. This should reflect in the latest extraction of the 2020-08-15 cord version.